rectools/models/sasrec.py

```diff
@@ -582,14 +605,19 @@ def __init__(
         loss: str = "softmax",
         verbose: int = 0,
         cpu_n_threads: int = 0,
+        deterministic: bool = False,
+        trainer: tp.Optional[Trainer] = None,
         transformer_layers_type: tp.Type[TransformerLayersBase] = SasRecTransformerLayers,  # SASRec authors net
         item_net_type: tp.Type[ItemNetBase] = IdEmbeddingsItemNet,  # item embeddings on ids
+        data_preparator_type: tp.Type[SessionEncoderDataPreparatorBase] = SasRecDataPreparator,
+        lightning_module_type: tp.Type[SessionEncoderLightningModuleBase] = SessionEncoderLightningModule,
+        pos_encoding_type: tp.Type[PositionalEncodingBase] = LearnableInversePositionalEncoding,
```

Comment:
Let's sort all of the arguments to follow clear logic: all net configuration arguments should be together, then learning configuration args. All of the optional `_type` args should be sorted too. Let's clean this a bit.

Reply:
fixed